ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDAmountRow.java

```diff
@@ -3,11 +3,9 @@
 import lombok.Builder;
 import lombok.Value;
 
-import java.math.BigDecimal;
-
 @Value
 @Builder
 public class CCDAmountRow {
     private String reason;
-    private BigDecimal amount;
+    private String amount;
```

Comment:
Changing a numeric field to String will not be a proper solution. The requirement is to store all of them from Pound to Pennies. So the correct change will be from BigDecimal to BigInteger.

By this way we know for sure these fields are numeric and cant have any string values.

Its better to conserve the field type (Numeric) and dont agree to changing to String.

Reply:
Please read the PR description: 

As per CCD Data formats:
https://github.com/hmcts/ccd-data-store-api/blob/master/docs/api/case-data.md#moneygbp
```
A MoneyGBP value is a string representing a monetary value in pence.
```

Reply:
we keep the object  type what ccd suggests, So I don't see any issue in changing to String

Reply:
@dharmendrak we should not code our domain for what external team ask for. There are ways to overcome this. We will discuss this soon.